hack/run-e2e-tests.sh

```diff
@@ -30,6 +40,11 @@ then
         | go-junit-report -set-exit-code \
         > reports/kudo_e2e_test_report.xml
 
+    ./bin/kubectl-kudo test --config kudo-upgrade-test.yaml 2>&1 \
```

Comment:
I mean, this is basically what i'm doing here. 

The wget is outside of kuttl, but thinking about it, it would make sense to put the logic of the current platform in a bash script and call that from within kuttl. I'll probably change a bit there, good ideas.

Apart from that... I didn't want to create a separate CI step, they're basically e2e tests, that's why they ended up in the run-e2e-tests.sh